jetson/nav/stateMachine.hpp

```diff
@@ -11,10 +11,11 @@
 #include "obstacle_avoidance/simpleAvoidance.hpp"
 #include "rover.hpp"
 #include "search/searchStateMachine.hpp"
-
+#include "recovery.hpp"
 
 using namespace rover_msgs;
 using namespace std::chrono_literals;
+using duration =  std::chrono::duration<int>;
```

Comment:
also generally people use `long long` as the `Rep` template parameter as it has higher precision. I think also clocks use nanoseconds under the hood so those can rack up fast and may not be held well by an `int`

Reply:
Would also recommend using steady clock for this since you are measuring deltas and dont care about absolute time. Steady clock is guaranteed never to go backwards..

In this particular case the time sync is not a huge concern since the granularity you need is on the order of seconds, but it's good practice.

Reply:
it has now been changed to steady clock and long long as Rep

Reply:
Nice, the `using` aliases are good too

Reply:
lgtm